Compatibility with React 17 and 18

Is there any plans to do this package compatible with React 17 and 18?

package.json contains "^16.4.2" React version as a dependency, And it worked 3 months ago with React 17. But Node moved their LTS Docker Alpine version to 16 and after that versions became incompatible.

Log from installation is below:

```
Status: Downloaded newer image for node:lts-alpine
npm notice 
npm notice New minor version of npm available! 8.11.0 -> 8.16.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.16.0>
npm notice Run `npm install -g npm@8.16.0` to update!
npm notice 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: app@1.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.4.2" from react-dadata-box@1.3.6
npm ERR! node_modules/react-dadata-box
npm ERR!   react-dadata-box@"^1.3.6" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Thank you!

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-dadata-box",
-  "version": "1.3.6",
+  "version": "1.3.7",
   "description": "Компонент для работы с DaData",
   "main": "dist/index.js",
   "style": "dist/index.css",

